local/compose/create.go

```diff
@@ -335,6 +368,15 @@ func getVolumesFrom(project *types.Project, volumesFrom []string) ([]string, []s
 
 }
 
+func getNetworkFrom(networkFrom string) string {
+	baseService := ""
+	if strings.HasPrefix(networkFrom, "service:") {
+		_, _ = fmt.Sscanf(networkFrom, "service:%s", &baseService)
```

Comment:
use of Sscanf is overkill, can just truncate string by `[len(servicePrefix):]`

Reply:
Done